server/services/database-service.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../models')
+
+module.exports = {
+  isConnected: async function () {
+    try {
+      await db.sequelize.authenticate()
```

Comment:
It looks like this re-authenticates each time, which isn't true - it actually runs `SELECT 1+1 AS result`.
Might be worth adding a comment to say that it's a lightweight check.

Reply:
I was a bit hesitant to do so as I'm not keen on documenting another library's API in our repo, but I've added just enough information to get the point across without having to worry too much if they change it.

Reply:
I also expanded the readme to mention the types of check the readiness probe does so I've added this as a line to that.